engine/run.ts

```diff
@@ -17,7 +17,10 @@ class Run {
             this.parseArgs();
             this.parsePlaybooks();
             this.parseEnvironments();
-            let entries = this.filterEnv(Array.from(this.environments.entries()))
+            let entries = this.filterEnv(Array.from(this.environments.entries()));
+            if(!this.args.has("debug")) {
+                console.debug = function(){}
```

Comment:
Can't we set some tag somewhere. I would find this better than override functions

Reply:
@SchettlerKoehler

Reply:
Yes, this would also be possible, but this is not the easiest solution, because you have to add a logger class that wraps the checking of this tag. To have the logging consistent you then have to change all the logging to use the new class. Currently, console is something like our logger, and overwriting functions of console to disable the output of some messages is a pattern in JS. I am not saying it is the best pattern but as the intention of this solution was to have something that can be implemented fast to debug an existing error I would go for this solution and think about if we need something better.